src/sass/partials/components/_forms.scss

```diff
@@ -24,6 +24,34 @@
     }
   }
 
+  &--add {
+    @include button(setting-get('primary accent color'), setting-get('primary accent color'), setting-get('primary color'), setting-get('primary accent color'), setting-get('primary color'), setting-get('primary color'), setting-get('primary color'));
+
+    background-image: url('/images/add.svg');
```

Comment:
If you have problems with that, extend https://github.com/punchcard-cms/punchcard/blob/master/lib/init/views.js like we did before